infer_structcol/model.py

```diff
@@ -12,6 +12,10 @@
 # define limits of validity for the MC scattering model
 min_phi = 0.35
 max_phi = 0.73
+min_radius = 70. # in nm
```

Comment:
edit: I see the explanation in model.ipynb that addresses this. It might be worth adding comments in the code so it is clear why these values were chosen.